### PR TITLE
feat: Replace signedMessage column for signature

### PR DESCRIPTION
**Pull request by juanmahidalgo**

### CODE REVIEW

File: src/Collection/Collection.router.ts [hunk 1]
```diff
@@ -241,70 +245,76 @@ export class CollectionRouter extends Router {
...
+      if (isTPCollection(dbCollection)) {
+        const itemIds = server.extractFromReq<string[]>(req, 'itemIds')
+        const dbItems = await Item.findByIds(itemIds)
+        result = await this.service.publishTPCollection(
+          dbCollection,
+          dbItems,
+          server.extractFromReq(req, 'signedMessage'),
+          server.extractFromReq(req, 'signature'),
+          server.extractFromReq(req, 'qty'),
+          server.extractFromReq(req, 'salt')
```
Comment (nicosantangelo) [inline src/Collection/Collection.router.ts:271]: just a question: Maybe this should be a single object? both for the client hitting the API but for this method's signature. I'm not sure about naming it tho. I was thinking of `cheque` but not sure.
Not blocking the merge, could be another PR

**Review by nicosantangelo — APPROVED** [review]
Comment but LGTM

File: src/Collection/Collection.router.ts (hunk 1 above)
Comment (juanmahidalgo) [reply src/Collection/Collection.router.ts]: I like the idea! Updated!
`cheque` added to https://github.com/decentraland/builder/pull/1820 as well

**Review by nicosantangelo — APPROVED** [review]
